Doc/library/re.rst

```diff
@@ -786,13 +786,17 @@ form.
 
 .. function:: escape(string)
 
-   Escape all the characters in pattern except ASCII letters, numbers and ``'_'``.
+   Escape special characters in a string.
```

Comment:
The normal way to write this would be "escape special characters in *pattern*", since *pattern* is the formal argument name in the source.  That is, I agree with your previous comment that string should be replaced by pattern.  The existing docs even use 'pattern' in the body of the text, so I'm not sure how we ended up with 'string' as the formal argument name in the docs.